src/index.ts

```diff
@@ -13,7 +13,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-module.exports.ClientFactory = require("./components/client-factory");
+export * from "./components/client-factory";
```

Comment:
Do I need to do this as well in #185?

Reply:
Probably yup, do the tests pass without it?